src/content/about/marketing.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Marketing activities
+description: Accepted practices for corporate involvement in the nf-core community
+---
+
+This page is intended for anyone coming to the Nextflow and nf-core community when representing a company or corporate interest.
+
+Firstly - welcome! We are an open and inclusive community and encourage everyone to be involved. Here we try to set out some guidelines to act as guard-rails for activity, both for the benefit of community members but also so that you feel comfortable that you are interacting with the community in acceptable ways.
+
+## Code of conduct
+
+The [code of conduct](/code_of_conduct/) applies to everyone active in the community.
+Two points are particularly worth highlighting, which are not allowed:
+
+- Sending unsolicited direct-messages or emails to community members on Slack / other platforms
+- Posting in nf-core spaces specifically to criticise or detract from Nextflow and nf-core, and to promote alternatives
+
+Contravening these, or any of the behaviours listed in the code of conduct, will likely result in an immediate ban.
```

Comment:
maybe a link to the code of conduct page or a snippet embedding instead of the actual rules makes it easier to keep changes in sync?

Reply:
The `[code of conduct](/code_of_conduct/)` is a link to the code of conduct, no?

Reply:
yes, but I would only point to them without copy-pasting them here.

Reply:
The code of conduct is very rarely updated, so I'm not too worried about this. It's also very long - hence I wanted to pull out the two lines in the document that were most relevant. I'm not sure that the risk of them getting out of sync is large enough to warrant not highlighting them here.